packages/sanity/src/core/components/StatusButton.tsx

```diff
@@ -70,31 +59,19 @@ export const StatusButton = forwardRef(function StatusButton(
 
   return (
     <Tooltip placement="bottom" {...tooltip} content={tooltipContent} disabled={!label} portal>
-      <div>
+      <ButtonWrapper>
```

Comment:
This component has been adapted to match with the new facelift UI.
Uses the new button, but adds a wrapper with position relative to position the `<Dot/>`
Added shadow to the span wrapper to show the positioning of the Dot
<img width="116" alt="Screenshot 2023-11-03 at 15 46 33" src="https://github.com/sanity-io/sanity/assets/46196328/6b01c1da-8597-4ddd-a8e6-ba43c13fe704">